### PR TITLE
tools: API bench supports etcd

### LINKED ISSUE #7703: Make bench tools more powerful
## Enhancement Task
To simulate the cases better, we should support more features for our bench tools, such as changing the bench parameters dynamically, etc.

### PR DESCRIPTION
**Pull request by CabinfeverB**
### What problem does this PR solve?

Issue Number: Close #7703

### What is changed and how does it work?

```commit-message
```

### Check List

Tests

- Manual test (add detailed scripts or steps below)

Code changes

- Has the configuration change
- Has HTTP API interfaces changed (Don't forget to [add the declarative for the new API](https://github.com/tikv/pd/blob/master/docs/development.md#updating-api-documentation))
- Has persistent data change

Side effects

- Possible performance regression
- Increased code complexity
- Breaking backward compatibility

Related changes

- PR to update [`pingcap/docs`](https://github.com/pingcap/docs)/[`pingcap/docs-cn`](https://github.com/pingcap/docs-cn):
- PR to update [`pingcap/tiup`](https://github.com/pingcap/tiup):
- Need to cherry-pick to the release branch

### Release note

```release-note
None.
```

### CODE REVIEW

**Comment by rleungx** [comment]
CI failed

File: tools/pd-api-bench/cases/cases.go [hunk 1]
```diff
@@ -366,3 +378,86 @@ func generateKeyForSimulator(id int, keyLen int) []byte {
...
+			baseCase: &baseCase{
+				name: "Get",
+				cfg:  newConfig(),
+			},
+		}
+	}
+}
+
+func (c *getKV) Unary(ctx context.Context, cli *clientv3.Client) error {
+	_, err := cli.Get(ctx, "/test/0001", clientv3.WithPrefix())
```
Comment (rleungx) [inline tools/pd-api-bench/cases/cases.go:398]: I think the key can be cluster id or others that must exist in etcd.

Comment (CabinfeverB) [reply tools/pd-api-bench/cases/cases.go:398]: It's okay. But I also have an idea, which is to initialize some key values

Comment (rleungx) [reply tools/pd-api-bench/cases/cases.go:398]: sounds good to me

**Review by rleungx — APPROVED** [review]

**Pull request merged**